HarpGuru/HarpFace/HarpCell/components/index.test.tsx

```diff
@@ -10,9 +11,12 @@ import { DisplayModes } from '../../../types'
 
 import { HarpCell } from './index'
 
+jest.mock('reactn')
+const mockUseGlobal = useGlobal as jest.Mock
```

Comment:
It would be good to come up with some kind of resolution to this requirement. Is this a problem with my dev environment setup?

Reply:
https://github.com/js-jslog/harpnative/issues/46 raised